modulus/utils/sdf.py

```diff
@@ -127,3 +128,66 @@ def signed_distance_field(
         return (sdf, sdf_hit_point_id)
     else:
         return sdf
+
+
+def sdf_to_stl(
```

Comment:
In my opinion this can live in the `modulus/utils/mesh` directory, and `sdf.py` should only contain the main modules for computing sdf and its derivates.

Reply:
Sounds good. I can move this.

Reply:
Moved